go/private/context.bzl

```diff
@@ -533,6 +527,12 @@ def go_context(
         cc_toolchain_files = depset()
         cgo_tools = None
 
+    # Path mapping can't map the values of environment variables, so we pass GOROOT to the action
+    # via an argument instead in builder_args. We need to drop it from the environment to get cache
+    # hits across different configurations since the stdlib path typically contains a Bazel
+    # configuration segment.
+    env_for_path_mapping = {k: v for k, v in env.items() if k != "GOROOT"}
```

Comment:
Sorry for the back and forth, but could you inline this all the way down in line 590? That should prevent similar issues from popping up in the future.

Reply:
Yes of course.